apheleia.el

```diff
@@ -519,24 +583,39 @@ symbols (matched against `major-mode' with `derived-mode-p') or
 strings (matched against value of variable `buffer-file-name'
 with `string-match-p'), and the values are symbols with entries
 in `apheleia-formatters' (or equivalently, they are allowed
-values for `apheleia-formatter'). Earlier entries take precedence
-over later ones.
+values for `apheleia-formatter'). Values can be a list of such
+symnols causing each formatter in the list to be called one after
+the other (with the output of the previous formatter).
+Earlier entries in this variable take precedence over later ones.
 
 Be careful when writing regexps to include \"\\'\" and to escape
 \"\\.\" in order to properly match a file extension. For example,
 to match \".jsx\" files you might use \"\\.jsx\\'\"."
   :type '(alist
-          :key-type symbol
-          :value-type symbol))
+          :key-type
+          (choice (symbol :tag "Major mode")
+                  (string :tag "Buffer name regexp"))
+          :value-type
+          (choice (symbol :tag "Formatter")
+                  (repeat
+                   (symbol :tag "Formatter")))))
 
 (defvar-local apheleia-formatter nil
   "Name of formatter to use in current buffer, a symbol or nil.
 If non-nil, then `apheleia-formatters' should have a matching
 entry. This overrides `apheleia-mode-alist'.")
 
-(defun apheleia--get-formatter-command (&optional interactive)
+(defmacro apheleia--ensure-list (arg)
```

Comment:
Does this need to be a macro? It seems like it would function identically as, excuse the pun, a function.